contrib/nydusify/pkg/parser/parser.go

```diff
@@ -118,7 +118,7 @@ func (parser *Parser) pullIndex(ctx context.Context, desc *ocispec.Descriptor) (
 }
 
 func (parser *Parser) parseImage(
-	ctx context.Context, desc *ocispec.Descriptor, onlyManifest *ocispec.Manifest,
+	ctx context.Context, desc *ocispec.Descriptor, onlyManifest *ocispec.Manifest, ignoreArch bool,
```

Comment:
There is no way for `nydusify` to specify `ignoreArch`?

Reply:
How is it going to be used? Please add an example in the docs.

Reply:
Updated comment. No way to specify. For a single manifest image, it's an automatic behavior to ignore the image arch, attempt to continue converting, and give a warning message. For the manifest list, nydusify selects host specified arch to convert, but the user still can specify an interesting platform (by --platform), the ultimate goal is that nydusify can convert any arch's image on any arch's host.